package protocal.log;


enum SystemType{
  CLIENTLOG = 1;
}